docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md

```diff
@@ -0,0 +1,21 @@
+# Other settings
+
+This topic will describe other settings in Nebula Dashboard.
+
+- Information: It shows cluster name, creates time, and creator.
+
+- Unbind: Unbind a cluster and remove its information from the platform, doing nothing to the cluster's services and data.
+
+  !!! note
+
+        When unbinding a cluster, you need to enter the cluster name to unbind.
```

Comment:
Before unbinding a cluster, enter the name of the cluster first.

Reply:
What abt "To unbind a cluster, enter the cluster name first"?

Reply:
> What abt "To unbind a cluster, enter the cluster name first"?

sure, perfect